### PR TITLE
Day11 : Adding Lecture Notes for Day 11

### PR DESCRIPTION
**Pull request by glendalyys**
In this file, you will find a markdown file that contains all the notes from Lecture Day 11. I've organized it by subject covered during the class day and included an objective of the class.

### CODE REVIEW

**Comment by kevinaangstadt** [comment]
please resolve the conflicts

**Comment by kevinaangstadt** [comment]
@jaxlub please review this PR in the comments

File: Lecture11.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
++ **Unit Testing** is when you test  indivual chunks of source code to determine that it works in isolation from the rest of the program. Some adavntages of this testing metric is as follows:
+* since it is in small chunks it's easier to understand what is going on in that specfic section.
+* we test features in isolation so it can be easier to locate a bug.
++ because it is small testing becomes faster.
+
+*** 
++ building on this idea, we can begin to discuss something called: **Test Driven Development**
+**Test Driven Development** is when you write the test file before you even write the new feature. Which works great in conversation with unit testing. For example:
++ Write a unit test for a new feature 
+    + _your test will fail at this point
```
Comment (jaxlub) [inline Lecture11.md:15]: The "_" appear to be a type and appears in the md file as -_your test will fail at this point

**Comment by jaxlub** [comment]
Also looks like there is a merge conflict to fix. In addition the file name does not match the style guide. Should be something like Day11Notes.md.

**Comment by kevinaangstadt** [comment]
@glendalyys please make the requested revisions and resolve the merge conflicts

**Pull request merged**